packages/flutter/test/material/text_field_focus_test.dart

```diff
@@ -548,6 +563,63 @@ void main() {
     expect(focusNodeA.hasFocus, false);
     expect(focusNodeB.hasFocus, true);
   }, variant: TargetPlatformVariant.desktop());
+
+  testWidgets('TextFormField gets blurred when tapping outside of the widget on mobile browsers', (WidgetTester tester) async {
```

Comment:
Can you also write a test to verify that clicking with a mouse does *not* unfocus the field?

Reply:
Also that reminds me, we try to just use the term "unfocus" and not "blur" in the framework.

Reply:
I've done it.

Reply:
> Can you also write a test to verify that clicking with a mouse does _not_ unfocus the field?

Clicking with a mouse will unfocus on desktops (supported at an early stage) and mobile browsers.